remix/app/tests/_.index.test.tsx

```diff
@@ -0,0 +1,9 @@
+import Index from "../routes/_index";
+import { render } from "@testing-library/react";
+
+describe("Test Root route", () => {
+  test("Should render Welcome Page", () => {
```

Comment:
Tests mus use the imperative voice (so no `should`):

```suggestion
  test("renders the Welcome Page", () => {
```